_includes/master/reqs-sys.md

```diff
@@ -31,19 +31,24 @@ to connect to, we provide instructions in the [installation documentation](./ins
 
 ## Network requirements
 
-{{site.prodname}} requires the network to allow the following types of traffic.
-
-| Traffic | Protocol | Port |
-| ------- | -------- | ---- |
-| BGP     | TCP      | 179  |
-| IPIP\*  | 4        | n/a  |
-
-\* {% if include.orch == "OpenStack" %} If your compute hosts connect directly
-   and don't use IPIP, you don't need to allow IPIP traffic. {% else %} Our
-   manifests enable IPIP by default.  If you disable IPIP, you won't need to
-   allow IPIP traffic. {% endif %} Refer to [Configuring
-   IP-in-IP](../../usage/configuration/ip-in-ip) for more information.
-
-> **Tip**: On GCE, you can allow this traffic using firewall rules. In AWS, use
-> EC2 security group rules.
-{: .alert .alert-success}
+Ensure that your hosts and firewalls allow the following traffic.
+
+| Configuration                                                                         | Host                | Connection type | Port/protocol                                                 |
+|---------------------------------------------------------------------------------------|---------------------|-----------------|---------------------------------------------------------------|
+| {{site.prodname}} networking                                                          | All                 | Bidirectional   | TCP 179                                                       |
+| {{site.prodname}} networking in IP-in-IP mode (default mode)                          | All                 | Bidirectional   | IP-in-IP, often represented by its protocol number `4`        |
+{%- if include.orch == "Kubernetes" %}
+| flannel networking (VXLAN)                                                            | All                 | Bidirectional   | UDP 4789                                                      |
+| etcd datastore                                                                        | etcd hosts          | Incoming        | Often TCP 2379 but varies per etcd configuration              |
```

Comment:
The "Often port X" wording here (and for the k8s apiserver) is weird. It should just say something like "Default value of 2379, but can be configured differently"